Cargo.toml

```diff
@@ -10,6 +10,8 @@ members = [
   "vsi",
 ]
 
+resolver = "2"
```

Comment:
Nope, we actually need to specify this.

This is because we use workspaces as "virtual" workspaces: https://doc.rust-lang.org/cargo/reference/workspaces.html#virtual-workspace

Virtual workspaces should specify resolver versions: https://doc.rust-lang.org/cargo/reference/resolver.html#resolver-versions

I added this because while working on this I noticed [feature unification](https://doc.rust-lang.org/cargo/reference/features.html#feature-resolver-version-2) wasn't working as expected (iirc I had some build failures from platform specific features).

Reply:
Ahhh. Got it. Thanks!